services/buildbot/master/master.cfg

```diff
@@ -247,6 +249,51 @@ builders.append({
         dependencies=BASE_DEPENDENCIES + WINDOWS_DEPENDENCIES),
     'category': 'unsupported'})
 
+#
+# Windows
+#
+builders.append({
+    'name': 'win-64-py2.7',
+    'slavenames': ['rax-win2012r2-1'],
+    'factory': TwistedVirtualenvReactorsBuildFactory(
+        git_update, RemovePYCs=Win32RemovePYCs,
+        dependencies=(BASE_DEPENDENCIES + CEXT_DEPENDENCIES +
+                      EXTRA_DEPENDENCIES + WINDOWS_DEPENDENCIES),
+        python="C:\\Python27\\python.exe",
+        reactors=["select", "iocp"],
+        platform="windows",
+        uncleanWarnings=True,
+        ),
+    'category': 'unsupported',
+    })
+
+
+builders.append({
+    'name': 'win-64-py2.7-coverage',
+    'slavenames': ['rax-win2012r2-1'],
+    'factory': TwistedCoveragePyFactory(
+        source=git_update, RemovePYCs=Win32RemovePYCs,
+        dependencies=(BASE_DEPENDENCIES + CEXT_DEPENDENCIES +
+                      EXTRA_DEPENDENCIES + WINDOWS_DEPENDENCIES),
+        python="C:\\Python27\\python.exe", platform="windows",
+        buildID="windows64py27",
+        uncleanWarnings=True,
+        ),
+    'category': 'unsupported',
+    })
+
+
+builders.append({
```

Comment:
I think this will overwrite the currently uploaded wheels... could we disable this one for now?